Dockerfile

```diff
@@ -1,5 +1,4 @@
-# debian stretch has libjemalloc1 https://packages.debian.org/stretch/libjemalloc1
-FROM ruby:2.5-slim-stretch
+FROM ruby:2.6-slim-buster
```

Comment:
as noted in the comment - we want to keep jemalloc v1 around and that package is available under the debian stretch version 

jemalloc is used for controlling ram growth / memory allocations for our app (especially our worker nodes). 

I'd suggest not removing this and instead using the 2.6 stretch image https://hub.docker.com/_/ruby/tags?page=1&name=2.6-slim

related jemalloc PR https://github.com/zooniverse/panoptes/pull/3088